### PR TITLE
Always set both collate and collation to the same value

### LINKED ISSUE #4798: 'utf8mb4_unicode_ci' is not valid for CHARACTER SET 'utf8'
### Affected version(s)

4.13, 5.x

### Description

In #4592 we added `collation: utf8mb4_unicode_ci` to the config skeleton. However, many users have something like this in their `config.yaml`

```yaml
doctrine:
    dbal:
        connections:
            default:
                default_table_options:
                    charset: utf8
                    collate: utf8_unicode_ci
```

due to https://docs.contao.org/manual/en/installation/system-requirements/#mysql-minimum-requirements

The addition in the config skeleton in the manager-bundle will then create a mismatch between the `charset` and the `collation`, as the `charset` is set to `utf8`, but the `collation` is set to `utf8mb4_unicode_ci` - which will result in an

```
'utf8mb4_unicode_ci' is not valid for CHARACTER SET 'utf8'
```

error when updating the database in the Install Tool or via `contao:migrate`.

Can anyone think of a solution for this, other than updating the documentation and changing `collate` to `collation`? (I am not sure if _both_ are actually needed as the original related Doctrine issues are hard to follow.)

### PR DESCRIPTION
**Pull request by fritzmg**
Fixes #4798, follow up to #4592

This PR adds an additional extension config via the Contao Manager Plugin so that both the `collate` _and_ the `collation` options are always set to the same value - in this case whichever value came last in the collection of extension configs.

### CODE REVIEW

**Review by ausi — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @fritzmg.
